src/test/functional/pages/BasePage.ts

```diff
@@ -85,4 +122,36 @@ export class BasePage {
     MAINTENANCE = 'Maintenance',
     CLAIM = 'Claim',
     BUZZ = 'BUZZ',
+    ADD_EMPLOYEE = 'Add Employee'
+  }
+
+  export enum Labels {
```

Comment:
It's generally recommended to name enums in a singular way. So it would be `Label` instead of `Labels`.  This reflects that each constant within the enum (like USERNAME, PASSWORD, LOGIN, etc.) represents a single, distinct label. This singular naming convention makes the code more intuitive and easier to understand, especially for other developers who might work with it in the future.

Reply:
right when we use it lebel.specific_label then it is one lebel not more